src/pages/treasury.js

```diff
@@ -124,68 +124,23 @@ const AnyQuestions = () => {
   );
 };
 
-const TreasuryReportSelection = () => {
+const ReportSelection = ({ type, months, caption }) => {
```

Comment:
let's make it so that ReportSelection only accepts one parameter: `type`. 

the `months` variable is not necessary since you can directly access MONTHS at the top of the file. and then, the `caption` variable is not necessary to pass it as parameter, you can dynamically calculate it inside the component based on the `type` parameter you received.